### PR TITLE
small collection of bugfixed debugger hints and warnings

### PR DESCRIPTION
**Pull request by SJunkies**
**Summary**

Just a small collection of bugfixed debugger hints and warnings ...

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of web configuration, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing setups:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's body (e.g. `Fixes: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated (docs/docs/en)
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by Paulchen-Panther** [comment]
Hint: [Hier](https://lgtm.com/projects/g/hyperion-project/hyperion.ng/alerts/?mode=list) gibt es noch einige Fehler zu beseitigen.

**Comment by SJunkies** [comment]
> Hint: [Hier](https://lgtm.com/projects/g/hyperion-project/hyperion.ng/alerts/?mode=list) gibt es noch einige Fehler zu beseitigen.

Danke, habe versucht noch ein paar Sachen zu fixen, aber bei ein paar Alerts war ich mir nicht ganz sicher...

ein Paar lgtm fixes, sind auch schon im Entertainment-API PR mit drin.

**Comment by Paulchen-Panther** [comment]
Ich hab da noch was. 😁 
https://prnt.sc/slx3sc

**Comment by SJunkies** [comment]
> Ich hab da noch was. 😁 
> 
> https://prnt.sc/slx3sc

Heute nicht mehr 😂

**Comment by Paulchen-Panther** [comment]
Na gut dann morgen. 🤣

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ We provide a macOS Build but we can not support this.
 
 ## Documentation
 Covers these topics (WorkInProgress)
-- Installtion
+- Installation
```
Comment (Paulchen-Panther) [inline README.md:44]: :smiley:

**Pull request merged**